webapp/anomaly_handler.go

```diff
@@ -33,23 +33,23 @@ func anomalyHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	ctx := appengine.NewContext(r)
-	query := datastore.
-		NewQuery(metrics.GetDatastoreKindName(
-			metrics.PassRateMetadata{})).
-		Order("-StartTime").Limit(1)
-	var metadataSlice []metrics.PassRateMetadata
+	passRateType := metrics.GetDatastoreKindName(metrics.PassRateMetadata{})
+	query := datastore.NewQuery(passRateType).Order("-StartTime").Limit(1)
 
+	var metadataSlice []metrics.PassRateMetadata
 	if _, err := query.GetAll(ctx, &metadataSlice); err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 	if len(metadataSlice) != 1 {
-		http.Error(w, "No metrics runs found",
-			http.StatusInternalServerError)
+		http.Error(w, "No metrics runs found", http.StatusInternalServerError)
```

Comment:
This returns 500 while its counterpart in `interop_handler.go` returns 404. I suppose one needs to be changed to be consistent?

Reply:
Done.